tests/conftest.py

```diff
@@ -281,6 +271,22 @@ def _fc(request, fc_module, new_user=False):
     return fc_module
 
 
+def _port(ip=_LOCALHOST):
+    import random
+    from sirepo import const
+
+    r = range(const.TEST_PORT_START, const.TEST_PORT_END)
+    port = random.choice(r)
+    for _ in r:
```

Comment:
Simplify `for p in random.sample(r, 100):`

Then you don't need port calc below, and 100 is better than 1000 when searching for ports. If we can't find a port after 100, something is wrong.

Reply:
Cool, that's better
[c490615](https://github.com/radiasoft/sirepo/pull/4846/commits/c4906152e17b741ed638943314bf35aa7222c15d)

I also moved the range into `const` since we only use the start and end ports to make a range anyway.